socrates/lib/registration/index.js

```diff
@@ -21,22 +21,37 @@ var roomOptions = beans.get('roomOptions');
 
 var app = misc.expressAppIn(__dirname);
 
-function isRegistrationOpen() {
-  // we currently set this to false on production system, because this feature is still in development
-  // the conf variable is used as a backdoor for testing
-  return app.get('env') !== 'production' && !conf.get('registrationIsClosed');
+function isRegistrationOpen(registrationParam) {
+  return moment(conf.get('registrationOpensAt')).isBefore(moment())
```

Comment:
moment(conf.get('registrationOpensAt') würde ich gerne in eine Variable rausziehen, dann kannst du es in registrationOpensIn wiederverwenden